src/prefect/flows.py

```diff
@@ -1361,7 +1361,7 @@ def __call__(
         )
 
     @sync_compatible
-    async def visualize(self, *args, **kwargs):
+    async def visualize(self, *args: Any, **kwargs: Any):
```

Comment:
We can do this right?
```suggestion
    async def visualize(self, *args: "P.args", **kwargs: "P.kwargs"):
```